src/modules/navigation/tree/TreeView.tsx

```diff
@@ -100,6 +100,13 @@ const TreeView = ({
     ? getIdsFromPath(focusedItem.path)
     : defaultExpandedIds;
 
+  // need to filter the expandedIds to only include items that are present in the tree
+  // we should not include parents that are above the current player root
+  const availableItemIds = itemsToShow?.map(({ id: itemId }) => itemId);
+  const accessibleExpandedItems = expandedIds.filter((e) =>
```

Comment:
Maybe you want to check `availableItemIds`'s length so you don't loop on `expandedIds` for nothing?